web/src/types/post.type.ts

```diff
@@ -0,0 +1,11 @@
+export interface Post {
+  dateCreated: Date;
+  id: string;
+  prompt: string;
+  imageURL: string;
+  author: {
+    id: string;
+    name: string;
+  };
+  likes: [];
```

Comment:
> 

I agree. The API endpoint will need to be updated to reflect this. At this point I would wait for @aMarshmallows to get the search api up and running before changing the type again in case something else changes

Reply:
Ok, sounds good. If you add a `// FIXME` above this so we don't forget, I will approve this.